src/pages/events/fall-2021-election.astro

```diff
@@ -0,0 +1,52 @@
+---
+import { Markdown } from 'astro/components';
+import Main from '../../layouts/Main.astro';
+---
+<Main title="Fall 2021 Election">
+	<div class="markdown mb-4">
+		<Markdown>
+			# Fall 2021 Election
+
+			Welcome to the Fall 2021 Election for the MUN Computer Science Society!
+
+			<br />
+
+			We're following a somewhat similar setup to our [previous election](https://muncomputersciencesociety.github.io/election-fall-2020/),
+			with keeping the majority of the interactions online for the sake of safety and convenience; especially for people who have still not arrived into Newfoundland yet.
+
+			## Timeline
+
+			- **Monday, September 20th** applications for candidacy open, closing on **September 27th**.
+			- Campaigning for applicants starts **September 28th**, closing on **October the 3rd**.
+			- Results of election presented (sometime on) **Monday, October the 4th**.
+
+			<br />
+			---
+			<br />
+
+			# Applications for Candidacy (September 20th → September 27th)
+
+			[We've put together a short Google Form which you can access here](https://forms.gle/18qnc1NXthRp1Lmq8).
```

Comment:
```suggestion
			[Candidacy Application Form](https://forms.gle/18qnc1NXthRp1Lmq8).
```
Can this link describe what it is, rather than saying we've put together a short form?